Lambdas/GHHooks/EventHandler/Handlers/ReleaseMaker.swift

```diff
@@ -299,21 +297,51 @@ struct ReleaseMaker {
     }
 
     func getExistingContributorIDs() async throws -> Set<Int> {
+        var page = 1
+        var contributorIds: [Int] = []
+        /// Hack: Vapor has around this amount of contributors and we know it, so better
+        /// to reserve enough capacity for it up-front.
+        contributorIds.reserveCapacity(250)
+        while let contributors = try await self.getExistingContributorIDs(page: page) {
+            let ids = (consume contributors).compactMap(\.id)
```

Comment:
btw i don't think these `consume`s do much since the type of the variable is not `NonCopyable`. At least not based on the latest info i have. Maybe later they'll implement something to ensure e.g. COW is skipped when using these keywords.